contracts/dev-utils/contracts/src/LibAssetData.sol

```diff
@@ -113,6 +113,9 @@ contract LibAssetData {
 
                 // Scale total balance down by corresponding value in assetData
                 uint256 scaledBalance = totalBalance / tokenValues[i];
+                if (scaledBalance == 0) {
```

Comment:
Similar to below, the token value can also be zero if this is nested asset data (MAP).